markdown/what.md

```diff
@@ -1,23 +1,79 @@
-can span multiple paragraphs.
 
-This is another paragraph. It is separated from another by a blank line.
-However, if a paragraph followed by **two** blank lines creates a new slide:
+I work in Cleura in the education team, and what we do is that we run the learning platforms. Our platforms are based on Open edX, a free open source learning management system.
+Open edX platforms have biannual releases like OpenStack, following a naming convention of botanical tree names. So when I mention the names "Maple", "Lilac" I am reffering to specific releases of the Open edX platform.
+
+Starting from the Maple release of Open edX, the community-supported method of deploying the platform using Ansible playbooks was deprecated. Instead, the upstream decided to transition towards a containerized deployment approach.
+
+To adapt to this change, we made the switch from OpenStack Heat to Kubernetes on OpenStack Magnum. This shift allowed us to embrace containerization for deploying the Open edX platform, leveraging the benefits it provides in terms of scalability, flexibility, and ease of management.
+
+Alongside the deployment changes, we also had a transition in our CI-driven deployment approach. We moved from Gitlab CI to Zuul.
+ 
+This is a summary of the architecture before and after, which also discusses some of the issues we ran into along the way.
 
 
 ## Open edX
 
 <!-- Note -->
-These are the notes for the next slide.
+
+As mentioned earlier, our platform is built on Open edX. Now let me provide you with a concise overview of what Open edX is and its fundamental components.
+
+Harvard University and MIT collaborated to establish edX, a non-profit organization with a mission to promote an open-source vision for the future of education. In 2013, edX made its code open source, enabling universities, corporations, NGOs, government organizations, and entrepreneurs to utilize this powerful tool.
+
+The edX platform comprises of three main components. Firstly, the Learning Management System (LMS) serves as the application through which learners access and engage with the course materials. The LMS provides a user-friendly interface for learners to view and interact with the educational content.
+
+Secondly, the platform includes Studio, an advanced course creation tool designed for instructors. Studio empowers instructors to develop and design their courses with ease. This sophisticated tool simplifies the process of creating engaging and interactive learning experiences.
+
+Lastly, Django Admin panel, the control house for adminstration and management tasks. It facilitates platform-level changes such as setting permissions and managing data, providing administrators with the tools needed to effectively manage the platform.
+
+All the components within the edX platform are meticulously designed and offer intuitive interfaces, ensuring that users can navigate and utilize the tools effortlessly.
```

Comment:
This sounds like an advertising phrase. This talk is not an advertisement. You can probably just cut this sentence.